modular_skyrat/modules/customization/modules/language/language_holder.dm

```diff
@@ -20,3 +14,9 @@
 							/datum/language/skrell = list(LANGUAGE_ATOM),
 							/datum/language/slime = list(LANGUAGE_ATOM)
 	)
+
+/datum/language_holder/shadowpeople
+	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+								/datum/language/voltaic = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+							/datum/language/voltaic = list(LANGUAGE_ATOM))
```

Comment:
Spreading things out makes code a lot more readable, and adding a trailing comma means that it's easier to modify without affecting the git blame as much.
```suggestion
	understood_languages = list(
		/datum/language/common = list(LANGUAGE_ATOM),
		/datum/language/voltaic = list(LANGUAGE_ATOM),
	)
	spoken_languages = list(
		/datum/language/common = list(LANGUAGE_ATOM),
		/datum/language/voltaic = list(LANGUAGE_ATOM),
	)
```